tests/test_jinja_report.py

```diff
@@ -58,6 +70,59 @@ def test_jinja_report(self):
             '<html><body>\n First rec Second rec\nSome data\n</body></html>'
         )
 
+    @with_transaction()
+    def test_jinja_report_logo(self):
+
+        # TODO: Generate those records with factories when
+        # trytond_factories contain those factories
+        pool = Pool()
+        Attachment = pool.get('ir.attachment')
+        ActionReport = pool.get('ir.action.report')
+        User = pool.get('res.user')
+        Party = pool.get('party.party')
+
+        action_report, = ActionReport.create([{
+            'name': 'Test Report Logo',
+            'report': 'jinja_report/tests/report_template_logo.html',
+            'report_name': 'jinja.test_report_logo',
+            'model': 'test.model',
+            'extension': 'pdf',
+            'template_extension': 'html',
+        }])
+
+        # GIVEN
+        user = User(1)
+        # FIXME: Party factory has customer_payment_term subfactory
+        # that belongs to account module that is not installed
+        # in this test. The same happens with warehouse sufactory
+        # in User factory that depends on stock module.
+        party, = Party.create([{'name': 'Nobody'}])
+        company = trytond_factories.Company.create(party=party)
+        user.company = company
+        user.companies = [company]
+        user.save()
+
+        with open(os.getcwd() + '/tests/logo.png', 'rb') as file:
+            logo = file.read()
+
+        Attachment.create([{
+            'resource': company,
```

Comment:
If the attachment and the action report were created in a 'test.model' record, that would be a huge simplification of the test for the sake of readability instead of using 'res.user' and 'company.company'.

'test.many2one' and 'test.many2one_target' can be used if you desire to demonstrate the ability to get a target record attachment from a report on the origin model.